### PR TITLE
Feature create group design

### PR DESCRIPTION
**Pull request by HarishChoure**
*Linked Work Items*
https://internship-splitwise.atlassian.net/browse/IS-74
*Checklist*
- [x] Code adheres to *[Git and Pull Request Guidelines]*
- [x] Code adheres to *[Angular Guidelines]*
- [x] Code adheres to *[Front-end Checklist]*

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
 export class DashboardComponent {
-
-}
+  constructor(private router : Router){}
+  navigateSignup(){
+    this.router.navigateByUrl('friend/friend-add-edit')
```
Comment (krupa310) [inline src/app/dashboard/dashboard.component.ts]: Update route name

File: src/app/expense/expense.module.ts [hunk 2]
```diff
@@ -6,8 +6,8 @@ import { ExpenseListComponent } from './expense-list/expense-list.component';
 import { RouterModule,Routes } from '@angular/router';
 
 const expenseRoutes : Routes = [
-  {path:'eList', component :ExpenseListComponent},
-  {path:'eAddEdit', component :AddEditExpenseComponent}
+  {path:'expense-list', component :ExpenseListComponent},
+  {path:'expense-add-edit', component :AddEditExpenseComponent}
```
Comment (krupa310) [inline src/app/expense/expense.module.ts]: Update route name

File: src/app/friends/add-edit-friends/add-edit-friends.component.html [hunk 3]
```diff
@@ -1 +1,34 @@
...
+        Email address is required
+      </div>
+      <!-- Invite friend textarea-->
+      <div class="input-field">
+        <label for="message">Message:</label>
+        <textarea id="message" name="message" formGroupName="discription" placeholder="Discription"></textarea>
+      </div>
+      <!-- Invite friend button -->
+      <div class="button-design">
+       <div> <button [disabled]="inviteForm.invalid"type="submit">Send Invite</button></div>
```
Comment (krupa310) [inline src/app/friends/add-edit-friends/add-edit-friends.component.html:30]: Remove extra HTML tags

File: src/app/friends/add-edit-friends/add-edit-friends.component.scss [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+    // Input field design
+    input[type="text"],
+    input[type="email"],
+    textarea {
+      flex: 1;
+      padding: 10px;
+      border: none;
+      border-radius: 5px;
+      box-shadow: 0px 0px 5px;
```
Comment (krupa310) [inline src/app/friends/add-edit-friends/add-edit-friends.component.scss]: Use shorthand properties

Comment (HarishChoure) [reply src/app/friends/add-edit-friends/add-edit-friends.component.scss]: Di , I am confuse about the shorthand can you brief this.

Comment (krupa310) [reply src/app/friends/add-edit-friends/add-edit-friends.component.scss]: if we have to add padding/margin for all sides then we can add it like, margin:10px 5px. by adding this we can set top/bottom margin to 10px,and left right margin to 5px

**Pull request merged**